Domains/QuantityTakeoffsAspects/QuantityTakeoffsAspects.ecschema.xml

```diff
@@ -94,13 +97,21 @@
     <ECEntityClass typeName="FoundationAspect" description="An aspect that describes the type of structural foundation." >
         <BaseClass>QuantityTakeoffAspect</BaseClass>
         <ECCustomAttributes>
-            <ClassHasHandler xmlns="BisCore.01.00.00"/>
+            <Deprecated xmlns="CoreCustomAttributes.01.00.03">
+                <Description>Use classes from the StructuralPhysical schema instead.</Description>
+            </Deprecated>
         </ECCustomAttributes>
+        <!-- TODO: Nabil to make sure all PhysicalElement classes in StructuralPhysical have a corresponding PhysicalType subclass. -->
```

Comment:
rather than TODO in schema, should be an item in some backlog?

Reply:
Created an issue for the StructuralPhysical team.